### PR TITLE
Update `azure-template` to reflect py27 only updates

### PR DESCRIPTION
**Pull request by scbedd**
- remove `setup.cfg` setting universal package
- remove py27 classifiers from `setup.py`
- add `python_requires` to `setup.py`

@xiangyan99

### CODE REVIEW

File: sdk/template/azure-template/setup.py [hunk 1]
```diff
@@ -65,9 +63,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.10.0",
     ],
-    extras_require={
-        ":python_version<'3.0'": ["azure-nspkg"],
```
Comment (scbedd) [inline sdk/template/azure-template/setup.py:69]: `nspkgs` are no longer required in a py3 world.

**Review by xiangyan99 — APPROVED** [review]

File: sdk/template/azure-template/setup.py [hunk 2]
```diff
@@ -43,8 +43,6 @@
     classifiers=[
         "Development Status :: 4 - Beta",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
```
Comment (xiangyan99) [inline sdk/template/azure-template/setup.py:47]: Maybe we also want to add 3.10?

**Review by lmazuel — APPROVED** [review]
Approved, but please add 3.10

File: sdk/template/azure-template/setup.py (hunk 2 above)
Comment (lmazuel) [reply sdk/template/azure-template/setup.py:47]: Also add
Programming Language :: Python :: 3 :: Only

**Comment by xiangyan99** [comment]
Thank you! @scbedd

**Pull request merged**